### PR TITLE
Add OWNERS files for automatic labeling

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
Add OWNERS files to relevant parts of the repo to allow PROW to automatically label PRs that touch those directories. We can optionally add "approvers" which roughly correlate to technical owners if we wanted.

### CODE REVIEW

File: ci/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+labels: 
+  - "A: maintenance"
```
Comment (petemounce) [inline ci/OWNERS]: I recommend using `A/thing` - more concise, leaves room for hierarchy if you decide later you want that, and is parseable as if it were a file-path via various stdlib support.

File: workers/unity/Packages/com.improbable.gdk.mobile/OWNERS.meta [hunk 2]
```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```
Comment (petemounce) [inline workers/unity/Packages/com.improbable.gdk.mobile/OWNERS.meta]: What is this?

Comment (jamiebrynes7) [reply workers/unity/Packages/com.improbable.gdk.mobile/OWNERS.meta]: A Unity metadata file

File: ci/OWNERS (hunk 1 above)
Comment (jamiebrynes7) [reply ci/OWNERS]: Hmmm hierarchy is nice - but I find the `/` notation harder to read at a glance 😕 

What would be advantage of the latter?

**Pull request merged**